packages/salesforcedx-vscode-core/src/commands/forceApexExecute.ts

```diff
@@ -6,23 +6,97 @@
  */
 import { ExecuteService } from '@salesforce/apex-node';
 import { Connection } from '@salesforce/core';
+import {
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
 import {
   CancelResponse,
   ContinueResponse,
   ParametersGatherer
 } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import * as fs from 'fs';
+import * as path from 'path';
 import * as vscode from 'vscode';
 import { channelService } from '../channels';
 import { nls } from '../messages';
 import { notificationService } from '../notifications';
+import { sfdxCoreSettings } from '../settings';
 import { telemetryService } from '../telemetry';
-import { hasRootWorkspace } from '../util';
+import { getRootWorkspacePath, hasRootWorkspace } from '../util';
 import {
   ApexLibraryExecutor,
+  CommandletExecutor,
   SfdxCommandlet,
+  SfdxCommandletExecutor,
   SfdxWorkspaceChecker
 } from './util';
 
+export class ForceApexExecuteExecutor extends SfdxCommandletExecutor<{}> {
```

Comment:
Is this shown as new because we removed it in a previous PR ?

Reply:
Yeah we removed it in the one that introduced the apex library changes